src/System.Management.Automation/engine/Modules/ModuleIntrinsics.cs

```diff
@@ -33,17 +33,15 @@ public class ModuleIntrinsics
         [TraceSource("Modules", "Module loading and analysis")]
```

Comment:
@iSazonov, your last commit had 3 failures in `PowerShell-CI-static-analysis`
Verify Markdown Links.Verify links in /home/vsts/work/1/s/docs/learning-powershell/powershell-beginners-guide.md.http://powershell.com/cs/blogs/ebookv2/default.aspx should work
```powershell
retry of URL failed with error: 
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 111
111:                                 throw "retry of URL failed with error: $($_.Message)"

```
Verify Markdown Links.Verify links in /home/vsts/work/1/s/docs/learning-powershell/README.md.http://powershell.com/cs/blogs/ebookv2/default.aspx should work
```powershell
retry of URL failed with error: 
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 111
111:                                 throw "retry of URL failed with error: $($_.Message)"

```
Verify Markdown Links.Verify links in /home/vsts/work/1/s/docs/learning-powershell/README.md.http://powershell.com/cs/media/14/default.aspx should work
```powershell
retry of URL failed with error: 
at <ScriptBlock>, /home/vsts/work/1/s/test/common/markdown/markdown-link.tests.ps1: line 111
111:                                 throw "retry of URL failed with error: $($_.Message)"

```